deploy/scripts/lhexec

```diff
@@ -39,13 +39,15 @@ exec_command() {
     VOLUME_NAME=${1}
     COMMAND_ARGS="${@:2}"
 
+	LONGHORN_BIN_PATH="/engine-binaries/longhornio-longhorn-engine-master/longhorn"
```

Comment:
No, this will not work. The `longhornio-longhorn-engine-master` will only exist if the manager is using `longhornio/longhorn-engine:master` image.

Reply:
fixed